packages/terra-toolbar/src/Toolbar.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './Toolbar.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Content alignment. Align to start, end, or center.
+   */
+  align: PropTypes.oneOf(['start', 'end', 'center']),
+  /**
+   * Items to be displayed in toolbar such as buttons, button groups, and links.
+   */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  align: 'start',
+};
+
+const Toolbar = ({
+  align, children, ...customProps
+}) => {
+  const toolbarClassNames = cx([
+    'toolbar',
+    `${align}-align`,
+    customProps.className,
+  ]);
+
+  const items = React.Children.map(children, item => (<div className={cx('item')}>{item}</div>));
```

Comment:
By wrapping the children, should we be expecting a <div> wrapper to make content in-line and this allows content to be stacked ?